test/integration/isomorphic-handler-test.js

```diff
@@ -280,7 +259,20 @@ describe("Isomorphic Handler", function () {
         .get("/http://localhost:3000")
         .expect(200)
         .then((res) => {
-          assert.equal("Prerender", res.text);
+          assert.equal(res.body.text, "Prerender");
+        })
+        .then(done, done)
+        .finally(() => {
+          newServer.close();
+        });
+    });
+
+    it("Should run prerender servre if i pass any other server top of it", function (done) {
+      supertest(newServer)
```

Comment:
This assertion is wrong. You should be doing something like this.https://stackoverflow.com/questions/46992795/testing-that-function-is-called-with-parameter-in-a-promise